kokoro/README.md

```diff
@@ -0,0 +1,22 @@
+# Kokoro CI jobs and testing scripts
+
+Firebase uses a Google-internal continuous integration tool called "Kokoro"
+(a.k.a "internal CI") for running majority of its open source tests. This
+directory contains the external part of kokoro test job configurations (the
+actual job definitions live in Google's internal monorepo) and the shell
+scripts that act as entry points to execute the actual tests.
+
+## Kokoro job structure
+
+Each job name should follow the format `product_target_method_xcodeversion.cfg`
+(i.e. `Core_macOS_xcodebuild_Xcode10_1.cfg`). The Xcode version may be omitted
```

Comment:
This was something I came up with by squishing the travis build preview parameters together into a snake case identifier, so I'm ok with changing it.

Omitting the default Xcode version seems like a good way to reduce visual clutter. Kokoro's docs mention that each xcode should be explicitly selected to prevent builds breaking when kokoro updates their default version and we'd have to rename all default Xcode builds every time the default version of Xcode changed, but this is not a huge amount of work and could probably be automated.